### PR TITLE
Tutorial Page and related features

### PR DESCRIPTION
**Pull request by tzimnick**

### CODE REVIEW

File: static/style.css
Comment (weiraven) [inline static/style.css:1]: Please move tutorial's page styling to tutorial's page html file. style.css are only stylings that should apply to the entire website. We also have a carousel on the homepage that carousel general stylings could conflict with.

**Review by weiraven — CHANGES_REQUESTED** [review]
Please move tutorials only stylings to tutorials page.

**Review by weiraven — APPROVED** [review]
Approving because we don't have time but tutorials-only stylings also do not belong on a style sheet that applies to the entire website.

**Review by ChaChang-Byte — APPROVED** [review]
shouldn't mess with most of other people's code

**Pull request merged**